### PR TITLE
Update release notes

### PR DESCRIPTION
**Pull request by Amir-61**
* Some cleanup and fixed a typo in release notes

### CODE REVIEW

File: 3.0-RELEASE-NOTES.md [hunk 1]
```diff
@@ -24,15 +24,15 @@ loopback-datasource-juggler from your dependencies.
...
+We consider Bluebird API as part of LoopBack API from now on,
 you are welcome to use any Bluebird-specific methods in your applications.
 
 If you are using LoopBack with a custom promise implementation provided
 via `global.Promise`,
 you will have to check all places where you are using non-standard promise API
 and update them to use Bluebird API instead.
 
-Please see [Related code change](https://github.com/strongloop/loopback/pull/1896) here.
+Please see [Related code change](https://github.com/strongloop/loopback/pull/1896).
```
Comment (superkhau) [inline 3.0-RELEASE-NOTES.md]: lowercase R for related, change->changes

File: 3.0-RELEASE-NOTES.md [hunk 2]
```diff
@@ -62,7 +62,7 @@ methods: {
 }
 ```
 
-Please see [related code change](https://github.com/strongloop/loopback/pull/2174) here.
+Please see [related code change](https://github.com/strongloop/loopback/pull/2174).
```
Comment (superkhau) [inline 3.0-RELEASE-NOTES.md]: ditto for plural change

File: 3.0-RELEASE-NOTES.md [hunk 3]
```diff
@@ -72,7 +72,7 @@ Use PersistedModel to report change-tracking errors via the existing method
 PersistedModel.handleChangeError. This method can be customized on a per-model basis to
 provide different error handling.
 
-Please see [related code change](https://github.com/strongloop/loopback/pull/2308) here.
+Please see [related code change](https://github.com/strongloop/loopback/pull/2308).
```
Comment (superkhau) [inline 3.0-RELEASE-NOTES.md]: ditto for plural change

File: 3.0-RELEASE-NOTES.md [hunk 4]
```diff
@@ -98,7 +98,7 @@ Developers that are relying on these properties, can redefine them in `user.json
 }
 ```
 
-Please see [Related code change](https://github.com/strongloop/loopback/pull/2299) here.
+Please see [Related code change](https://github.com/strongloop/loopback/pull/2299).
```
Comment (superkhau) [inline 3.0-RELEASE-NOTES.md]: lowercase R for related, plural for change

File: 3.0-RELEASE-NOTES.md [hunk 5]
```diff
@@ -231,4 +231,4 @@ now removed in favor of promoting use of:
 - `app.registry.createModel(modelName, properties, options)` to create new model
 - `app.model(modelCtor, config)` to update existing model and attach it to app
 
-Please see [related code change](https://github.com/strongloop/loopback/pull/2401) here.
+Please see [related code change](https://github.com/strongloop/loopback/pull/2401).
```
Comment (superkhau) [inline 3.0-RELEASE-NOTES.md]: plural for change

**Comment by Amir-61** [comment]
@superkhau 

Thanks for the review... I addressed your concerns, and I'm going to land it now.

Thanks!

**Pull request merged**